src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/lecture-unit-management.route.ts

```diff
@@ -9,166 +9,81 @@ import { CreateTextUnitComponent } from 'app/lecture/lecture-unit/lecture-unit-m
 import { EditTextUnitComponent } from 'app/lecture/lecture-unit/lecture-unit-management/edit-text-unit/edit-text-unit.component';
 import { CreateVideoUnitComponent } from 'app/lecture/lecture-unit/lecture-unit-management/create-video-unit/create-video-unit.component';
 import { EditVideoUnitComponent } from 'app/lecture/lecture-unit/lecture-unit-management/edit-video-unit/edit-video-unit.component';
-import { CourseResolve } from 'app/course/manage/course-management.route';
-import { LectureResolve } from 'app/lecture/lecture.route';
 
-// TODO: The /edit routes have no crumb because individual attachments have no route => make child of overview
 export const lectureUnitRoute: Routes = [
```

Comment:
Is it necessary to have two separate `Route` objects with `path: 'unit-management'` here, or can you merge the attributes of both into one?

Reply:
That is indeed on purpose, I just forgot to include a comment as per #2631. Fixed by adding a comment.